Userland/Libraries/LibWeb/Painting/ImagePaintable.h

```diff
@@ -34,6 +34,8 @@ class ImagePaintable final
 
     bool m_renders_as_alt_text { false };
     String m_alt_text;
+
+    Layout::ImageProvider const& m_image_provider;
```

Comment:
The things that inherit from `ImageProvider` are JS heap allocated, so can't the object get GC'd out from under us? Seems `ImageBox` has the same issue if that's the case

Reply:
I added missing visits in both ImageBox and ImagePaintable.